### PR TITLE
feat: add interface related and testing configuration file

### PR DESCRIPTION
**Pull request by blackcat-118**
## Description
- Users can set the WiFi interface name in the sec.conf file instead of modifying the source code. 
- This PR also includes an example configuration file for development testing.

## TODO
- [x] delete redundant configuration file
- [x] update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,5 @@
 # EAP 5G
 wpa_supplicant modified version for EAP 5G
```
Comment (andy89923) [inline README.md]:
```suggestion
# TNGFUE
wpa_supplicant modified version for EAP 5G
```

Comment (andy89923) [reply README.md]: and move the line `wpa_supplicant modified version for EAP 5G` in top of `wpa_supplicant/README`

**Review by andy89923 — COMMENTED** [review]
- If `wpa_supplicant/defconfig` does not need, remove it
- Delete `wpa_supplicant/ChangeLog`
- Delete `wpa_supplicant/eap_testing.txt`
- Delete `README` (Not README.md) 
- Delete `.DS_Store`

File: README.md [hunk 2]
```diff
@@ -1,38 +1,5 @@
...
-> sudo ./wpa_supplicant -c ../wpa_supplicant.conf -i <wifi interface name> -dd
-        
-// 2. wired conncetion
-> sudo ./wpa_supplicant -c ../wpa_supplicant.conf -i <wired interface name> -D wired -dd
-```
-3. For wireless connection, turn on wifi connection **after** the command executed.
-## File changed
-- ```src/eap_peer/eap_vendor_test.c```: The main extension for 5G data
-- ```wpa_supplicant/.config```: Open the setting for ```VENDOR-TEST```
-- ```wpa_supplicant/Makefile```: let the vendor type suppport key encryption
```
Comment (andy89923) [inline README.md]: These should be found somewhere in document or README. (Even it's not support for now).

Comment (blackcat-118) [reply README.md]: I think the `File changed` section is no longer needed.

Comment (blackcat-118) [reply README.md]: Should I leave a comment that the wired option has not been tested?

Comment (andy89923) [reply README.md]: If the option is working(even if it's not working normally), leave it with a comment; otherwise removed.

Comment (blackcat-118) [reply README.md]: fixed.

**Review by andy89923 — APPROVED** [review]
LGTM

**Pull request merged**